app/assets/stylesheets/helper.scss

```
@@ -0,0 +1,39 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+@import 'color.scss';
+
+/* sidebar */
+aside{
+  section.user_info{
```

```suggestion
  section.user_info {
```
tương tự các chỗ khác